markovs_household/data/read_probabilities.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pandas as pd
+import numpy as np
+from markovs_household.config.config import Configuration
+
+
+def calculate_probabilities(df, type):
+    config = Configuration.parse_config(r"tests\input\config.yaml")

```

Comment:
die config liest du einmal zu Simulationsbeginn ein, dann kannst du einfach die Attribute der Konfiguration die du in der Methode brauchst als Argument mitgeben